pkg/apis/cluster/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -2,7 +2,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2021 The KCP Authors.
+Copyright 2022 The KCP Authors.
```

Comment:
What caused this?

Reply:
I've run the codegen manually to avoid the CI codegen-verify failing due to the boilerplate changed.

Reply:
Wait, so with 2021 our CI fails? @ncdc is that your understanding of how this should work? I thought only *new* files needed to be updated?

Reply:
Yup. As codegen-verify runs, it updates the boilerplate and detects a diff. I dropped the codegen, check the result :)

Reply:
@stevekuznetsov no, we shouldn't be verifying years in generated code headers.